src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -243,25 +247,25 @@ private function displayGeneralJson(SymfonyStyle $io, $filter)
 
     private function getLoaderPaths(string $name = null): array
     {
-        /** @var FilesystemLoader $loader */
-        $loader = $this->twig->getLoader();
         $loaderPaths = [];
-        $namespaces = $loader->getNamespaces();
-        if (null !== $name) {
-            $namespace = $this->parseTemplateName($name)[0];
-            $namespaces = array_intersect([$namespace], $namespaces);
-        }
+        foreach ($this->getFilesystemLoaders() as $loader) {
+            $namespaces = $loader->getNamespaces();
+            if (null !== $name) {
+                $namespace = $this->parseTemplateName($name)[0];
+                $namespaces = array_intersect([$namespace], $namespaces);
+            }
 
-        foreach ($namespaces as $namespace) {
-            $paths = array_map([$this, 'getRelativePath'], $loader->getPaths($namespace));
+            foreach ($namespaces as $namespace) {
+                $paths = array_map([$this, 'getRelativePath'], $loader->getPaths($namespace));
 
-            if (FilesystemLoader::MAIN_NAMESPACE === $namespace) {
-                $namespace = '(None)';
-            } else {
-                $namespace = '@'.$namespace;
-            }
+                if (FilesystemLoader::MAIN_NAMESPACE === $namespace) {
+                    $namespace = '(None)';
+                } else {
+                    $namespace = '@'.$namespace;
+                }
 
-            $loaderPaths[$namespace] = $paths;
+                $loaderPaths[$namespace] = $paths;
```

Comment:
this is broken if multiple loaders are providing paths for the same namespace (which is totally valid), as you would override the paths instead of merging them.

Reply:
I'll send a PR to fix that, thank you!